kamelets/aws-ddb-sink.kamelet.yaml

```diff
@@ -73,6 +73,7 @@ spec:
         title: Operation
         description: "The operation to perform. The options are PutItem, UpdateItem, or DeleteItem."
```

Comment:
In the documentation this will appear as enum, so we could remove them from the description

Reply:
Done.